config-object-store/src/main/java/org/hypertrace/config/objectstore/IdentifiedFilterPushedDownObjectStore.java

```diff
@@ -26,13 +31,23 @@ protected IdentifiedFilterPushedDownObjectStore(
         resourceName,
         configChangeEventGenerator,
         clientConfig);
+    this.configServiceBlockingStub = configServiceBlockingStub;
+    this.resourceName = resourceName;
```

Comment:
So this is why you initially had that method it in the `IdentifiedObjectStore` I'm guessing. Rather than shadowing these variables in the subclass, could we just set them to package private in `IdentifiedObjectStore`?

Reply:
yes, may be i should have explained to you then itself.

Reply:
Sorry I should be clearer with my comments - by setting "them" package private I meant the declarations of the variables from `IdentifiedObjectStore` that you were shadowing in this subclass. 

I would still keep the filter impl in this subclass because that's actually the method doing what this class is built/named for -  pushing down the filter.

Reply:
I have kept methods here and added variables also here. Make variables package private in `IdentifiedObjectStore` leading to errors in ContextuallyIdentifiedObjectStore.
`Cannot reference 'IdentifiedObjectStore.configServiceBlockingStub' before superclass constructor is called`

Reply:
That's just a name clash since that inner class is referencing unrelated parent class variables
![image](https://github.com/user-attachments/assets/5809e4f0-2d71-40fc-8f3e-8c0b479a5031)